docs/source/getting-started.md

```diff
@@ -0,0 +1,201 @@
+
+#### Runtime APIs
+Models can be loaded and run from Swift as follows:
+```swift
+// TODO Code sample
+```
+
+For more information on iOS integration, including an API reference, logging setup, and building from source, see [Using ExecuTorch on iOS](/TODO.md).
+
+### C++
+ExecuTorch provides C++ APIs, which can be used to target embedded or mobile devices. The C++ APIs provide a greater level of control compared to other language bindings, allowing for advanced memory management, data loading, and platform integration.
+
+#### Installation
+CMake is the preferred build system for the ExecuTorch C++ runtime. To use with CMake, clone the ExecuTorch repository as a subdirectory of your project, and use CMake's `add_subdirectory("executorch")` to include the dependency. The `executorch` target, as well as kernel and backend targets will be made available to link against. The runtime can also be built standalone to support diverse toolchains. See [Using ExecuTorch with C++](/TODO.md) for a detailed description of build integration, targets, and cross compilation.
+
+```
+git clone -b release/0.5 https://github.com/pytorch/executorch.git
+```
+```python
+# CMakeLists.txt
+add_subdirectory("executorch")
+...
+target_link_libraries(
+  my_target
+  PRIVATE executorch
+          executorch_module_static
+          executorch_tensor
+          optimized_native_cpu_ops_lib
+          xnnpack_backend)
+```
+
+#### Runtime APIs
+Both high-level and low-level C++ APIs are provided. The low-level APIs are platform independent, do not dynamically allocate memory, and are most suitable for resource-constrained embedded systems. The high-level APIs are provided as a convenience wrapper around the lower-level APIs, and make use of dynamic memory allocation and standard library constructs to reduce verbosity.
+
+ExecuTorch uses CMake for native builds. Integration is typically done by cloning the ExecuTorch repository and using CMake add_subdirectory to add the dependency.
+
+Loading and running a model using the high-level API can be done as follows:
+```cpp
+#include <executorch/extension/module/module.h>
+#include <executorch/extension/tensor/tensor.h>
+
+using namespace ::executorch::extension;
+
+// Load the model.
+Module module("/path/to/model.pte");
+
+// Create an input tensor.
+float input[1 * 3 * 256 * 256];
+auto tensor = from_blob(input, {1, 3, 256, 256});
+
+// Perform an inference.
+const auto result = module.forward(tensor);
+
+if (result.ok()) {
+  // Retrieve the output data.
+  const auto output = result->at(0).toTensor().const_data_ptr<float>();
+}
+```
+
+For more information on the C++ APIs, see [Using ExecuTorch with C++](/TODO.md).
```

Comment:
Use these links for now 

https://pytorch.org/executorch/stable/extension-module.html
https://pytorch.org/executorch/stable/extension-tensor.html